etc/concepts.tsv

```diff
@@ -0,0 +1,336 @@
+King	King	1508	KING
+MaleSlave	Male Slave		
+FemaleSlave	Female Slave		
+Four	Four	1500	FOUR
+Medicine	Medicine	1372	MEDICINE
+Hair	Hair	1040	HAIR
+Forehead	Forehead	123	FOREHEAD
+Nose	Nose	1221	NOSE
+Eye	Eye	1248	EYE
+Ear	Ear	1247	EAR
+Mouth	Mouth	674	MOUTH
+Five	Five	493	FIVE
+Tooth	Tooth	1380	TOOTH
+Tongue	Tongue	1205	TONGUE
+Throat	Throat	1346	THROAT
+Gullet	Gullet		
+Neck	Neck	1333	NECK
+Shoulder	Shoulder	1482	SHOULDER
+Arm	Arm	1673	ARM
+ArmBetweenShoulderandElbow	Arm Between Shoulder and Elbow		
+ArmBetweenElbowandWrist	Arm Between Elbow and Wrist		
+Leg	Leg	1297	LEG
+OuterHandorHand	Outer Hand, or Hand		
+InnerHand	Inner Hand		
+FootSole	Foot-Sole	2666	SOLE (FOOT)
+Elbow	Elbow	981	ELBOW
+Chest	Chest	1592	CHEST
+FemaleBreast	Female Breast	1402	BREAST
+Belly	Belly	1251	BELLY
+Thigh	Thigh	800	THIGH
+Knee	Knee	1371	KNEE
+Skin	Skin	763	SKIN
+Bone	Bone	1394	BONE
+Vein	Vein	1924	VEIN
+Blood	Blood	946	BLOOD
+Hat	Hat	771	HAT
+Cap	Cap	1288	CAP
+Waistcloth	Waist-cloth		
+House	House	1252	HOUSE
+Doorway	Door-way		
+Bed	Bed	1663	BED
+Mat	Mat	195	MAT
+Knife	Knife	1352	KNIFE
+Spoon	Spoon	1378	SPOON
+FatherMyFatherThyFather	Father (My Father, Thy Father)	1217	FATHER
+MotherMyMotherThyMother	Mother (My Mother, Thy Mother)	1216	MOTHER
+ElderBrotherMyThy	Elder Brother (My, Thy)	1759	OLDER BROTHER
+YoungerBrotherMyThy	Younger Brother (My, Thy)	1760	YOUNGER BROTHER
+ElderSister	Elder Sister	1758	OLDER SISTER
+Doctor	Doctor	597	PHYSICIAN
+Head	Head	1256	HEAD
+Face	Face	1560	FACE
+FootorInstepoftheFoot	Foot, or Instep of the Foot		
+Finger	Finger	1303	FINGER
+Navel	Navel	1838	NAVEL
+NailofFingerandToe	Nail (of Finger and Toe)	1896	FINGERNAIL OR TOENAIL
+Shirt	Shirt	1622	SHIRT
+Door	Door	1567	DOOR
+ArmletorBracelet	Armlet or Bracelet		
+Spear	Spear	945	SPEAR
+Sun	Sun	1343	SUN
+Night	Night	1233	NIGHT
+Coal	Coal	2658	COAL
+Drum	Drum	908	DRUM
+Firewood	Firewood	10	FIREWOOD
+Walkingstick	Walking-stick	1296	WALKING STICK
+Leaf	Leaf	628	LEAF
+Groundnut	Ground-nut	595	GROUNDNUTS
+Milk	Milk	635	MILK
+Cat	Cat	1208	CAT
+Pig	Pig	1337	PIG
+Bat	Bat	1793	BAT
+FowlHen	Fowl (Hen)		
+Fish	Fish	227	FISH
+Mosquito	Mosquito	1509	MOSQUITO
+Bee	Bee	665	BEE
+Honey	Honey	942	HONEY
+Chamelion	Chamelion	1555	CHAMELEON
+Lizardthecommonone	Lizard (the common one)	632	LIZARD
+Black	Black	163	BLACK
+Rich	Rich	712	RICH
+Igo	I go	695	GO
+Istop	I stop	2880	HALT (STOP)
+Isitdown	I sit down	1649	SIT DOWN
+Ibreathe	I breathe	1407	BREATHE
+Isnore	I snore	1983	SNORE
+Ilaugh	I laugh	1355	LAUGH
+Ifall	I fall	2894	TUMBLE (FALL DOWN)
+Itake	I take	1749	TAKE
+Ilovethee	I love thee	923	LOVE
+Icatchafish	I catch a fish	2638	FISHING
+Ibreakastick	I break a stick	2558	BREAK (CLEAVE)
+Bow	Bow	994	BOW
+Arrow	Arrow	977	ARROW
+Quiver	Quiver	995	QUIVER
+Greegree	Greegree	2804	AMULET
+Sacrifice	Sacrifice	1103	SACRIFICE
+Hell	Hell	878	HELL
+Waterfresh	Water (fresh)		
+Gold	Gold	1369	GOLD
+Book	Book	963	BOOK
+NewMoon	New Moon	3691	NEW MOON
+Soap	Soap	788	SOAP
+Needle	Needle	1382	NEEDLE (FOR SEWING)
```

Comment:
Not sure but I would leave it unmapped due to the lack of specificity. This is unlikely to be the case, but it still can refer to a needle of a pine tree or the like.